hydra-redis/test/querying_test.py

```diff
@@ -76,44 +81,20 @@ def ControllerLogCollectionmemberTest(self):
             check_data[0]) in data1 and str(
             check_data[1]) in data1 and str(
                 check_data[2]) in data1:
-            return True
+            self.assertTrue(True)
         else:
-            return False
+            self.assertTrue(False)
+        
 
-    def DatastreamCollectionmemberTest(self):
+    def test_5_DatastreamCollectionmember(self):
         """Test for all datastream with Drone ID 2"""
         check_data = ['/api/DatastreamCollection/19']
         query = "show DatastreamCollection members"
         data = query_facades.user_query(query)
         # Here are find the datastream only for those which have DroneID 2.
         query = "show DroneID 2 and type Datastream"
         data = query_facades.user_query(query)
-        if data == check_data:
-            return True
-        else:
-            return False
-
-
-class TestQueryingMechanism(unittest.TestCase):
-    query_test = Tests()
-
-    def test_1_endpoint(self):
-        self.assertTrue(self.query_test.endpointTest())
-
-    def test_2_classendpoint(self):
-        self.assertTrue(self.query_test.classendpointTest())
-
-    def test_3_collectionendpoint(self):
-        self.assertTrue(self.query_test.collectionendpointTest())
-
-    def test_4_CommandCollectionmember(self):
-        self.assertTrue(self.query_test.CommandCollectionmemberTest())
-
-    def test_5_ControllerLogCollectionmember(self):
-        self.assertTrue(self.query_test.ControllerLogCollectionmemberTest())
-
-    def test_6_DatastreamCollectionmember(self):
-        self.assertTrue(self.query_test.DatastreamCollectionmemberTest())
+        self.assertEqual(data,check_data)
 
 
 if __name__ == "__main__":
```

Comment:
Sandeep you're supposed to initialize the graph and everything in the `setUp` function. It is meant to do the same. Rest everything looks fine for now. Create an issue for the `and or` query part then we can merge this.

Reply:
@xadahiya  If we initialize the graph every time for every test with `setup` method then there will be data stored multiple times in Redis server(because Redis server is running independently to the files). And It'll provide duplicate data for which we already have an issue #24 ( I am working on it). And due to duplicate data test cases will fail.